computers/README.txt

```diff
@@ -47,3 +47,5 @@ Thanks to all the people in the forums and the #minetest IRC channel for
 
 See also:
 http://minetest.net/
+
+license WTFPL https://github.com/minetest-mods/computer/blob/master/README.txt#L36-L38
```

Comment:
Should use a permalink here.

```suggestion
License WTFPL [https://github.com/minetest-mods/computer/blob/master/README.txt](https://github.com/minetest-mods/computer/blob/2f74fa2857d9d212b0207c78fc1c103f63672c09/README.txt#L36-L38)
```

Reply:
Hmm, on second thought, maybe it should be relicensed? 🤔

Reply:
sure, MIT seems like a good option

Reply:
https://github.com/mt-mods/home_workshop_modpack/pull/3/commits/6f9a27ff05c73df310bdd053e694acebd742607e resolved. formated as md since you suggested that even though it is a .txt file

Reply:
Ah, didn't notice it was `.txt`